src/Dialect/ONNX/AdditionalONNXOps.td.inc

```diff
@@ -0,0 +1,24 @@
+def ONNXCustomOp:ONNX_Op<"Custom",
```

Comment:
.inc files are reserved for auto-generated files. We used to have an additional file for the extra ops not in the standard. Looks like we need it again

Reply:
done